### ISSUE TITLE
University of Colorado Clean up

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#7775]
Use this template when you discover two agents that you believe represent the same person or organization.

**Agent that should remain after the merge**
This agent should represent the same person or organization as the agent below, but be of higher quality (more complete name, or includes additional information such as dates, relationships, addresses). Navigate to the agent page, copy the url and paste it below:

[University of Colorado Boulder](https://arctos.database.museum/agent/21348494)

**Agent that will no longer remain after the merge**
This agent should be of lower quality than the agent into which it will be merged. Navigate to the agent page, copy the url and paste it below:

[University of Colorado](https://arctos.database.museum/agent/21253845)

**Reason for merge**
Please provide the reason you believe that these two agents represent the same person or organization.

Aren't these the same thing? The other option is to change [University of Colorado](https://arctos.database.museum/agent/21253845)

to

University of Colorado Boulder, Biology Department

If we don't know WHICH University of Colorado, then [University of Colorado](https://arctos.database.museum/agent/21253845) should NOT be a verified agent.

Please add this issue to the Agents Project

@ArctosDB/agents-committee @ebraker @acdoll @jldunnum @wellerjes

**Comment by wellerjes** [comment]
I cleaned up the ones associated with CHAS; most were actually Boulder.

**Comment by ebraker** [comment]
I am fine with merging 'University of Colorado' to 'University of Colorado Boulder'. Technically there is a University of Colorado Denver campus and a University of Colorado Colorado Springs campus, but all my collections refer to UC Boulder.

I would not advocate 

> University of Colorado Boulder, Biology Department

We have an Evolutionary Biology Department (EBIO), that was historically called Evolutionary and Population Biology (EPOB). Perhaps long ago it was referred to as the Biology department but it would sort of be a dead-end agent (btw UCM is not affiliated with EBIO, it is its own institute).

**Comment by acdoll** [comment]
I'll fix ours, but it is still linked to several CHAS and MSB accessions.

**Comment by Jegelewicz** [comment]
> but it is still linked to several CHAS and MSB accessions.

@wellerjes @campmlc @jldunnum

**Comment by jldunnum** [comment]
Out of country until 29 May but maybe Mariel can deal with this.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Jegelewicz** [comment]
@jldunnum no rush, it is just a duplicate agent.

**Comment by wellerjes** [comment]
It was connected to our accession records, I think I've updated all of them.

**Comment by dustymc** [comment]
>automate

I don't think so, there needs to be a human involved, still not clear to me that these have the same intention (when was that remark added, and who noticed?). I can update stuff, just let me know exactly what and if anything needs more investigation....
